osquery/tables/system/windows/logged_in_users.cpp

```diff
@@ -66,7 +67,7 @@ QueryData genLoggedInUsers(QueryContext& context) {
               << ")";
       continue;
     }
-    auto wtsSession = (PWTSINFO)sessionInfo;
+    auto wtsSession = reinterpret_cast<WTSINFOA*>(sessionInfo);
```

Comment:
Could `wtsSessin` be const?

Reply:
I think so, yeah. I'll have to get rid of the subsequent assignments to `nullptr`, but that's not a big deal!

Reply:
Done!